docs/source/examples.rst

```diff
@@ -19,76 +19,97 @@ Tidy3D Basics
     notebooks/StartHere
     notebooks/Simulation
     notebooks/Primer
+    notebooks/WebAPI
 
 Feature Tutorials
 -------------------
 
-Structures and Materials
-~~~~~~~~~~~~~~~~~~~~~~~~
+Mediums
+~~~~~~~
 
 .. toctree::
     :maxdepth: 1
 
-    notebooks/GDSImport
-    notebooks/STLImport
     notebooks/Dispersion
     notebooks/Fitting
     notebooks/FullyAnisotropic
     notebooks/Gyrotropic
     notebooks/CustomMediumTutorial
+
+Structures
+~~~~~~~~~~
+
+.. toctree::
+    :maxdepth: 1
+
+    notebooks/GDSImport
+    notebooks/STLImport
     notebooks/SelfIntersectingPolyslab
+    notebooks/PICComponents
+    notebooks/PhotonicCrystalsComponents
 
-Grid and Boundary Conditions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Grid Specification
+~~~~~~~~~~~~~~~~~~
 
 .. toctree::
     :maxdepth: 1
 
-    notebooks/BoundaryConditions
     notebooks/AutoGrid
 
-Sources and Monitors
-~~~~~~~~~~~~~~~~~~~~
+Boundary Conditions
+~~~~~~~~~~~~~~~~~~~
 
 .. toctree::
     :maxdepth: 1
 
-    notebooks/ModalSourcesMonitors
-    notebooks/ModesBentAngled
-    notebooks/TFSF
-    notebooks/CustomFieldSource
+    notebooks/BoundaryConditions
 
-Running Simulations
-~~~~~~~~~~~~~~~~~~~
+Sources
+~~~~~~~
 
 .. toctree::
     :maxdepth: 1
 
-    notebooks/WebAPI
-    notebooks/ParameterScan
+    notebooks/ModalSourcesMonitors
+    notebooks/ModesBentAngled
+    notebooks/TFSF
+    notebooks/CustomFieldSource
 
-Visualization of Simulation and Monitor Data
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Monitors and Data Visualization
```

Comment:
maybe "Data Visualization and Postprocessing"?

Reply:
OK.. nice!